SharedPackages/BrowserServicesKit/Sources/BrowserServicesKit/Watchdog/Watchdog.swift

```diff
@@ -25,10 +25,21 @@ import os.log
 ///
 public final actor Watchdog {
     /// The current state of the main thread.
-    public enum HangState {
+    public enum HangState: CustomStringConvertible {
```

Comment:
You could do this instead:

```suggestion
    public enum HangState: String {
```

Then remove `description` and use `rawValue` instead :sunglasses: